reconcile/test/utils/unleash/test_unleash_client.py

```
@@ -9,7 +9,7 @@
 from reconcile.utils.unleash.client import (
     DisableClusterStrategy,
     EnableClusterStrategy,
-    _get_unleash_api_client,  # noqa: PLC2701
+    _get_unleash_api_client,  # ruff: ignore[import-private-name]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: import-private-name

Enable `lint.preview` to use rule names

(RUF102)